src/content/vintage/platform-overview/security/operational-layers/index.md

```diff
@@ -72,38 +71,26 @@ With a valid *jwt* token, received from your chosen identity provider, customers
 - *admin*
-    - full access, to the `cluster`, `node pool`, `appcatalogs` and `apps` resources of management cluster Kubernetes.
+    - full access, to the `cluster`, `node pool`, `appcatalogs` and `apps` resources of the management cluster.
     - includes *view* level access.
 
-### Giant Swarm REST API {#giant-swarm-api}
+There's a possibility for admins to create sub-roles to give other users within their organization access to Platform API features. As the API is a Kubernetes API, this is facilitated with standard RBAC role definitions.
 
-The [Giant Swarm REST API](/api/) is a customer facing API that is usually whitelisted for only a certain IP range within the customer's network. This layer covers the API itself, and its client manifestations in the form of the Giant Swarm Web UI and `gsctl` CLI.
+This enables use cases where the central platform team can give limited access to sub-teams like and observability team, but also go as far as to give end user developers access to certain resources and use the Platform API as the API to their own Developer Platform.
 
-On this layer there are two levels of access:
-
-#### 1. Giant Swarm REST API admin
-
-This access level is reserved for Giant Swarm operations and support personnel and like the above layers is facilitated by SSO with MFA.
-
-Admin users have access to all organizations and all clusters in the Giant Swarm installation.
-
-#### 2. Giant Swarm REST API user
-
-This is the standard type of Giant Swarm REST API user that is given out to DevOps/Operations personnel on the customer side. Usually that covers only few users that are tasked with cluster creation and management.
-
-Such users have access to all clusters in the organizations they belong to. They can create new clusters and organizations as well as manage or delete cluster and organization that they are part of. They can be considered multi-cluster admins.
+Additionally, the Platform API includes a concept called organizations, which are special namespaces that can hold clusters and their apps, and are used to create a level of multi-tenancy within the Platform API. Tenant admins can create such organizations and bind users and user groups to them, so that action of these users are limited to the resources in the organizations they are part of.
 
 ### User space {#userspace}
 
 The user space layer is defined as the layer pertaining to a single workload cluster Kubernetes API. Workload clusters are the Kubernetes clusters that run your workloads.
 
-Users on this level are either created by a Giant Swarm REST API user (in form of key pairs) or managed in an external identity provider (IdP), like [Azure Active Directory]({{< relref "/vintage/advanced/access-management/authentication-azure-ad" >}}) or any other OIDC-compliant IdP.
+Tenant admins have `cluster-admin` roles in all workload clusters of the installation.
```

Comment:
I was not super sure how that looks like in CAPI, do we bind them automatically? do they have access to a kubeconfig? cause the Rest API calls are not available, right?

Reply:
I would then also remove the two Rest API related Further readings

Reply:
ah, so it can still create certs like it used to? with custom org and email as username?

Reply:
It's complicated :smiling_face_with_tear:
- We do encourage customers to use our dex app with their IDP of choice on their WCs which works using OIDC in `kubectl gs login` but they need to set up their own RBAC to go with it
- By default `kubectl gs login --workload-cluster...` can be used to create client certificates for workload clusters. In vintage these certs are signed through `Vault` and in CAPI through the CA secret stored for each WC in the org namespace. (so yes, customers need access to the org namespace on the MC to log in)

Reply:
Yes. We never got around to introduce any rbac management for wcs sadly. :)

Reply:
all good, will try to adjust the wording to reflect